x-pack/plugins/canvas/public/components/app/index.js

```diff
@@ -51,10 +47,9 @@ const mapDispatchToProps = dispatch => ({
   setAppReady: basePath => async () => {
     try {
       // initialize the socket and interpreter
-      await createSocket(basePath);
       loadPrivateBrowserFunctions();
       await populateBrowserRegistries(types, basePath);
-      await initializeInterpreter();
+      await isInterpreterInitialized();
```

Comment:
The fact we're not doing anything with the result indicates this method is incorrectly named, (see above, as well).